api/include/opentelemetry/trace/noop.h

```diff
@@ -58,6 +59,14 @@ class OPENTELEMETRY_EXPORT NoopSpan final : public Span
                 const common::KeyValueIterable & /*attributes*/) noexcept override
   {}
 
+#if OPENTELEMETRY_ABI_VERSION_NO >= 2
+  void AddLink(const SpanContext & /* target */,
+               const common::KeyValueIterable & /* attrs */) noexcept override
+  {}
+
+  void AddLinks(const SpanContextKeyValueIterable & /* links */) noexcept override {}
+#endif
```

Comment:
nit - also add tests for this in noop_test.cc

Reply:
Implemented, see NoopTest.UseNoopTracersAbiv2.